src/ProjectFileTools/TextViewCreationListener.cs

```diff
@@ -37,6 +40,11 @@ public void VsTextViewCreated(IVsTextView textViewAdapter)
             CompletionController completion = new CompletionController(view, _completionBroker);
             textViewAdapter.AddCommandFilter(completion, out IOleCommandTarget completionNext);
             completion.Next = completionNext;
+
+            // Command Filter for GoToDefinition in .csproj files
+            GotoDefinitionController gotoDefinition = new GotoDefinitionController(view, _workspaceManager);
+            textViewAdapter.AddCommandFilter(gotoDefinition, out IOleCommandTarget gotoDefinitionNext);
+            gotoDefinition.Next = gotoDefinitionNext;
```

Comment:
This order of construct, register, set .Next should be encapsulated so that you can never construct a controller without .Next being set. I suggest privatizing the constructor of `GotoDefinitionController` and making a static function on it that takes in the `textViewAdapter` and performs the 3 steps at once. `GotoDefinitionController.CreateAndRegister(...)`